capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -178,6 +213,9 @@ public Object getRequestResult(HttpServletRequest request)
         String searchWord = request.getParameter("stringToSearchBy");
         return runSearchStudySetSqlQuery(pool, searchWord);
       }
+      else if(pathInfo.startsWith("/user")) {
+          return getStudySetFromUserId(pool, pathInfo);
```

Comment:
wrong indentation looks like; should be two spaces.

Reply:
Ran a code formatter on the file